### ISSUE TITLE
Console login locked out after switch to either ubuntu or debian console.

### ISSUE THREAD

**Issue by rtis** [rancher/os#1566]
**RancherOS Version: (ros os version)**
V0.7.1 (selected by "ros install cmd" after booting current ISO from flash drive)

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
Bare metal Intel NUC new i3.  32GB ram, 500GB ssd

Ran into this issue and wanted to check for possible solutions. After bringing up
ISO (latest available), I changed rancher account password to rancher - works as expected.
Created cloud-config.yml file w/ssh key and issued "sudo ros install -c cloud-config.yml
-d /dev/sda" command. Again, works as expected. Once system reboots, I issue
"sudo ros console switch debian" (or ubintu). Appears to pull correctly. Then perform
a "reboot". When the system comes back up, I cannot login from console.

Have seen a number of issues on this topic; but they appear to have been resolved.
Would like to move on to more interesting tests, but can't leave this issue without
resolving it.....any suggestions?

**Comment by joshwget** [comment]
@SvenDowideit This sounds very familiar. You fixed a similar issue recently, right?

**Comment by SvenDowideit** [comment]
mmm - I thought the thing i fixed was something that broke during 0.8 dev

**Comment by SvenDowideit** [comment]
@rtis are you able to ssh into the box using the ssh key?

**Comment by rtis** [comment]
Have tried that and no joy!  Have been googling for hints; but may be turning up issues
previously addressed.  Does swapping out the console change any of the OS state - like
/etc/passwd or .ssh contents?

Glad to try forcing load of a particular version.  Is there an easy way to tell
"ros install" which version to install?

rtisys

**Comment by SvenDowideit** [comment]
ok, so the last way to get into your existing install is to set a password of autologin at the grub boot screen 

the 2 things you want to append to the kernel parameters line is `autologin=ttyS0 rancher.password=rancher`

I'm really surprised this is happening on 0.7.1 - I'll go test on an aws vm though

**Comment by SvenDowideit** [comment]
so I followed along - boot 0.7.1, make a config.yml that has an ssh key, did `ros install -d /dev/sda -c config.yml`,  was not able to login, so i used `rancher.autologin=tty1` in grub to get me in - change the password, and then did a `ros console switch debian` - after reboot, i again couldn't login

I'll test to see if its this bad in 0.8.0-rc8 :/

switching from the default console to a persistent one shouldn't in my mind cause this type of thing - but there are some weird corners to it.

**Comment by rtis** [comment]
Thanks for following up on this....also glad to know I wasn't doing something
completely wrong.  To add one thing, after initial ISO boot but prior to issuing
ros install cmd, I changed rancher password to "rancher" then issued ros install.
I could log in at the console or ssh with that account/password after the system
rebooted.    It was only after I tried to switch
default console to debian or ubuntu that things fell apart.

rtis
